src/app/page.tsx

```diff
@@ -133,7 +136,7 @@ const Home = async ({ searchParams }: Props) => {
             <div className="flex justify-between mb-4">
               <div className="flex flex-col [@media(min-width:724px)]:flex-row gap-4 items-start sm:items-center">
                 <div className="flex gap-4 items-center">
-                  <div className="flex justify-center items-center h-[64px] w-[64px] rounded-2xl bg-[#FF7DCB]">
+                  <div className="flex justify-center items-center h-[64px] w-[64px] rounded-2xl bg-ocs-pink">
```

Comment:
If this is part of our palette, it should be defined in a theme config?`

Reply:
It is - in tailwind.config.js

I'm changing this to use the already existing pallette

Reply:
Sorry, I read this the same as here https://github.com/base-org/onchainsummer.xyz/pull/307/files#diff-73d7a23e5015801b9bcc9db601d6ec9594d3eb34e5bb23154e0ae4b0c30f1a3bR55. Should that be added as well?

Reply:
gotcha - yeah I'm reconciling those in this PR

https://github.com/base-org/onchainsummer.xyz/pull/308

And then I'm going to reconcile all the uses of `[#EFEFEF]` and `[#444]` in another PR upcoming